.github/workflows/fast-forward-manual.yml

```diff
@@ -14,4 +14,7 @@ on:
 jobs:
   call-ff:
     uses: ./.github/workflows/fast-forward-branch.yml
+    with:
+      ref: ${{ github.event.inputs.ref }}
+      to_branch: ${{ github.event.inputs.to_branch }}
```

Comment:
You removed `github.event` from other places. Why keep it here?

Reply:
That was the source of the problem. The Github event in this case is `workflow_dispatch`, that is, manually triggered, and `github.event.inputs` are the inputs sent when manually triggering the pipeline. Now check this link: https://stackoverflow.com/a/71458860 

This file `fast-forward-manual.yml` is geting some event inputs, but in the scheduled one `fast-forward-managed-services.yml` the 'event' is automatic and gets no inputs, those empty inputs where inherited by the reusable workflow.

So in `fast-forward-manual.yml` we want to get the inputs from the event. But in the reusable workflow `fast-forward-branch.yml` we want to get the inputs from `input` context, which contains the parameters sent to the workflow call.

Reply:
in stackoverflow it is `github.event.input.some_other_value`. That means `input`, not `inputs`. But given that worked before, I guess it is fine.

Reply:
The [contexts](https://docs.github.com/en/actions/learn-github-actions/contexts) are called `github.event.inputs` and `inputs` in the docs. The stackoverflow answer has a typo on it.